### PR TITLE
Feature/application info

### PR DESCRIPTION
**Pull request by AlexandrosMor**
Add application info for classic payment and checkout api call

### CODE REVIEW

File: Adyen/client.py [hunk 1]
```diff
@@ -87,8 +87,8 @@ def __init__(self, username=None, password=None, xapikey=None,
         self.skin_code = skin_code
         self.psp_list = []
         self.app_name = app_name
-        self.LIB_VERSION = "1.4.0"
-        self.USER_AGENT_SUFFIX = "adyen-python-api-library/"
+        self.LIB_VERSION = settings.LIB_VERSION
```
Comment (rkewlani) [inline Adyen/client.py]: do we need to assign settings.LIB_VERSION ? do we need self.LIB_VERSION ?

Comment (AlexandrosMor) [reply Adyen/client.py]: self.LIB_VERSION is the one that belong to the class. The settings.LIB_VERSION it is the one that we set as constant.

File: Adyen/client.py [hunk 2]
```diff
@@ -87,8 +87,8 @@ def __init__(self, username=None, password=None, xapikey=None,
...
+        self.USER_AGENT_SUFFIX = settings.LIB_NAME
```
[the remark this answers is not in the document]
Comment (AlexandrosMor) [reply Adyen/client.py]: ok It is updated with back slash

File: Adyen/client.py (hunk 1 above)
Comment (AlexandrosMor) [reply Adyen/client.py]: yes it is needed. It is used by the HTTPClientto create the full 
 self.user_agent = app_name + " " + user_agent_suffix + lib_version

**Pull request merged**